src/components/Scorecard.vue

```diff
@@ -361,7 +361,7 @@ export default defineComponent({
             return true;
           },
         })
-        .then((blob) => {
+        .then((blob: any) => {
```

Comment:
just don't assign `any` to every variable - find out what is the right type for them and use that. don't focus on just removing the error but on the right solution.